lib/similarity_utils.py

```diff
@@ -270,6 +271,7 @@ def get_ranking_new(data: pd.DataFrame, main_actor_id: str, ranking_length: int
     """metoda dla ramki danych, id aktora oraz (opcjonalnie) długości rankingu, zwraca listę id aktorów najbardziej
     podobnych do wybranego aktora;
     metoda jest przygotowana pod dane ze zbiorów treningowego, testowego i walidacyjnego"""
+    warnings.warn("Jest nowa funkcja get_ranking_for_actor_data_row")
```

Comment:
zmień nazwę funkcji przywoływaną w ostrzeżeniu, powinna to być już funkcja `get_ranking_for_actor_data_row_new`

Reply:
Zrobione